Add clearer docs about what both JS integrations support (#401)
appsignal/appsignal-docs: source/front-end/installation.html.md

Please follow the [installation guide](/guides/new-application.html) first, when adding a new application to AppSignal.

## Creating a Push API Key

> Before you begin, you'll need to locate your Push API key. Finding this is easy - look for the ["Push and deploy" section](https://appsignal.com/redirect-to/app?to=info) of your App settings page. You'll be able to find your API key under the "Front-end error monitoring". Once you have your key, follow the instructions under the [Installation](/front-end/installation.html) section to use it in your application.

Reviewer: ```suggestion
Before you begin, you'll need to find your Push API key on the ["Push and deploy" section](https://appsignal.com/redirect-to/app?to=info) of your App settings page. You'll be able to find your API key under the "Front-end error monitoring". Once you have your key, follow the instructions under the [Installation](/front-end/installation.html) section to use it in your application.
```
